crits/samples/handlers.py

```diff
@@ -1034,6 +1056,10 @@ def handle_file(filename, data, source, method='Generic', reference=None, relate
         sample = Sample()
         sample.filename = filename or md5_digest
         sample.md5 = md5_digest
+        sample.sha1 = sha1_digest
+        sample.sha256 = sha256_digest
+        sample.size = size
```

Comment:
This should be cast to an int to make sure it doesn't go in as a string (https://github.com/crits/crits/blob/master/crits/samples/sample.py#L106)